docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3179,14 +3179,17 @@ are explained in more depth in the table below.
   :header-rows: 0
 
   * - Noise Function Affected
-    - Additional Inputs 
+    - Additional Input
     - Default Input Value 
   * - Typographic Noise
     - Probability that a corrupted token contains the original token (e.g., if typically e -> r, the probability that e -> er)
     - 0.1 
   * - Age miswriting 
     - Possible perturbations of age (e.g., for [1, -1] and age of 7, the possible "incorrect" results will be 6 and 8)
     - [1, -1] 
+  * - Age miswriting
+    - Probabilities associated with possible age perturbations (must be same length as possible perturbations; probabilities must sum to 1)
```

Comment:
I think we should also allow passing `None` for the probabilities, in which case they default to the uniform probability measure over the choice of increments.